tests/_support/ea-server-mocker/js/ea-mocker.js

```diff
@@ -0,0 +1,71 @@
+(function ( $, undefined ) {
+	var formatJSONString = function ( text ) {
```

Comment:
Good to know; [WP Coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/) go for snakeCase in JS so I stuck to that.
Coding standards are my black beast.